### PR TITLE
Replaced the DropdownField component with the RadioInput component.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

Figma: https://www.figma.com/file/7TEcX7f6rw3Z0WYR4TpX3R/COVID-Symptom-Study-App-MASTER?node-id=11435%3A882
Notion: https://www.notion.so/joinzoe/017c3de591604a7b89d0b69e3c37b279?v=0133357df3ed42e1bd42c1e0d7afcf45&p=3e759567865e4cee98c12392fb4d076b

Screenshot:
Simulator Screen Shot - iPhone 11 - 2021-06-22 at 22 40 42

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if needed)

## Checklist (for reviewers)

- [x] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

_Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

**Review by juliatan — APPROVED** [review]

**Comment by selvach** [comment]
**Suggestions**

[1] "Choose one of these options" should not be a selectable option

Screenshot 2021-06-23 at 15 32 10

**Comment by aikewoody** [comment]
> **Suggestions**
> 
> [1] "Choose one of these options" should not be a selectable option
> 
> ### VaccineDetails Screen
> Screenshot 2021-06-23 at 15 32 10
> 
> ## AboutYou Screen (Profile Creation)
> Screenshot 2021-06-23 at 15 44 16
> 
> [2] Can you change the Text Font or contrast to be darker?

@selvach Good spot with `[1]`!

About `[2]` I agree the text is too light. It's a weird one because the color/font-family/font-weight/etc. is exactly the same as in the design on Figma but it still looks more light somehow

**Comment by aikewoody** [comment]
More dark font family (regular instead of light) for the text:
Simulator Screen Shot - iPhone 11 - 2021-06-23 at 17 44 57

File: src/features/patient/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export { default as AboutYouScreen } from './AboutYouScreen';
 export { NHSDetailsScreen } from './NHSDetailsScreen';
 export { NHSIntroScreen } from './NHSIntro';
-export { default as PreviousExposureScreen } from './PreviousExposure';
+export { default as PreviousExposureScreen } from './PreviousExposureScreen';
```
Comment (selvach) [inline src/features/patient/index.ts:4]: Thank you for tidying this up!

File: src/routes/index.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ import {
   JoinSchoolGroupScreen,
   JoinSchoolScreen,
   LoginScreen,
-  LongCovidQuestionPageOneScreen,
+  LongCovidQuestionScreen,
```
Comment (selvach) [inline src/routes/index.tsx:29]: Thank you for tidying this up!

**Review by selvach — APPROVED** [review]
Thank you for the code tidy as well as this lovely new design pattern for forms. It's a HUGE improvement!

**Pull request merged**